app/assets/stylesheets/_variables.scss

```diff
@@ -18,37 +12,23 @@ $header-line-height: 1.25;
 // Other Sizes
 $base-border-radius: 3px;
 $base-spacing: $base-line-height * 1em;
+$small-spacing: $base-spacing / 2;
 $base-z-index: 0;
 
 // Colors
-$blue: #477DCA;
+$blue: #477dca;
 $dark-gray: #333;
 $medium-gray: #999;
-$light-gray: #DDD;
-
-// Background Color
-$base-background-color: white;
```

Comment:
My point was if / when it's handy there isn't anything stoping you from putting it in. How often have you used an inset on a project?